Werewolf for Telegram/Werewolf Node/Helpers/Settings.cs

```diff
@@ -105,6 +105,7 @@ public static int
             ChemistSuccessChance = 50,
             GraveDiggerConversionChance = 30,
             AugurConversionChance = 40,
+            ScapeGoatConversionChance = 80,
```

Comment:
Really? This poor role has resistance against the cult?
Doesn't feel very logical to me...